src/ui/common/src/components/pages/workflows/components/RunWorkflowDialog.tsx

```diff
@@ -0,0 +1,276 @@
+import {
```

Comment:
Should there be some code get removed? Looks like this component is a current feature but I'm having a hard time figuring where they come from

Reply:
Yes, this code was removed from `workflowHeader.tsx`. It's reflected in the PR, but it's wrapped up in a bunch of other code changes in that file.